docker/Dockerfile

```diff
@@ -1,22 +1,36 @@
 FROM ubuntu:22.04
 
-RUN apt update \
-    && apt-get install -y \
+# install dependencies 
+
+RUN apt update
+RUN apt-get install -y \
     g++ \
     libatlas-base-dev \
     python3 \
     python3-dev \
     python3-pip \
     python3-venv \
-    swig
+    swig \
+    # required for wheel-less libsbml installation 
+    cmake
```

Comment:
No longer required? (also `\` won't work with the comment in the following line)